recipes/jwt-cpp/all/test_package/conanfile.py

```diff
@@ -0,0 +1,25 @@
+from conans import ConanFile, CMake, tools
+
+import os
+
+class JsonCppTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake_find_package"
+
+    @property
+    def _is_multi_configuration(self):
+        cmake = CMake(self)
+        return cmake.is_multi_configuration
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            bin_path = "example"
+            if self._is_multi_configuration:
+                bin_path = os.path.join(str(self.settings.build_type), bin_path)
+            self.output.info(bin_path)
+            self.run(bin_path, run_environment=True)
```

Comment:
I copied pasted this from another project, I am not sure why the executable path is incorrect, it works on my windows system

Any help would be appreciated

Reply:
See https://github.com/prince-chrismc/conan-center-index/pull/1